src/legacy/core_plugins/kibana/public/management/sections/index_patterns/create_index_pattern_wizard/create_index_pattern_wizard.test.tsx

```diff
@@ -167,36 +143,29 @@ describe('CreateIndexPatternWizard', () => {
   });
 
   it('invokes the provided services when creating an index pattern', async () => {
-    const get = jest.fn();
-    const set = jest.fn();
     const create = jest.fn().mockImplementation(() => 'id');
     const clear = jest.fn();
-    const changeUrl = jest.fn();
+    services.indexPatterns.clearCache = clear;
+    const indexPattern = ({
+      id: '1',
+      title: 'my-fake-index-pattern',
+      timeFieldName: 'timestamp',
+      fields: [],
+      create,
+    } as unknown) as IndexPattern;
```

Comment:
Please use IIndexPattern

Reply:
I have done it because services.indexPatterns.make should return IndexPattern. If I use IIndexPattern, I get an error that the "make" method should return IndexPattern.